src/frontend/javascript/init.js

```diff
@@ -44,3 +44,11 @@ if ($tooltips) {
     new Tooltip($tooltips[i]).init();
   };
 }
+
+var $tabs = document.querySelectorAll('[data-module="govuk-tabs"]');
```

Comment:
Do you know if this will break our other tabs?

`data-module="govuk-tabs"` is used here: https://github.com/alphagov/paas-admin/blob/693ba3e7049c441a0f84ace817bfbcf88e39ef1b/src/components/services/views.tsx#L80

Reply:
Nevermind, just noticed you're fixing it.

Reply:
yep. it wouldn't break, just throw a js error.